Dockerfile

```diff
@@ -15,12 +15,27 @@ RUN cargo build --release --locked --no-default-features
 RUN rm -f target/release/deps/git_cliff*
 
 FROM debian:buster-slim as runner
+
+# Everything inside this container will be explicitly mounted by the end user,
+# so we can sidestep some Git security restrictions. This app recommends
+# mounting data to /app, but this *can* be changed externally and *will* be
+# changed when run by GitHub Actions, so we need to cover our bases.
+RUN echo -e '[safe]\n\tdirectory = *' > /etc/gitconfig
```

Comment:
The contents of this file is the following in the container I built locally:

```
root@040df823c1dc:/app# cat /etc/gitconfig
-e [safe]
	directory = *
```

Thus, `git-cliff` fails:

```
root@040df823c1dc:/app# git-cliff
 WARN  git_cliff > "cliff.toml" is not found, using the default configuration.
 ERROR git_cliff > Git error: `failed to parse config file: invalid configuration key (in /etc/gitconfig:1); class=Config (7)`
```